src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--29">
+      <div class="calendar__day day day--1 day--start-day--sun"></div>
+      <div class="calendar__day day day--2"></div>
```

Comment:
try making the same thing, but without `day` and `day--#` classes. use a for loop instead.